packages/schema-generator/API.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@aws-amplify/schema-generator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export class DatabaseConnectError extends Error {
+    constructor(code: string, errorno: string);
+    // (undocumented)
+    readonly code: string;
+    // (undocumented)
+    readonly errorno: string;
+}
+
+// @public (undocumented)
+export type MySQLEngine = 'mysql';
+
+// @public
+export const parseDatabaseUrl: (databaseUrl: string) => SQLDataSourceConfig;
+
+// @public (undocumented)
+export type PostgresSQLEngine = 'postgresql';
+
+// @public
+export class SchemaGenerator {
+    // (undocumented)
+    generate: (props: SchemaGeneratorConfig) => Promise<void>;
+}
+
+// @public (undocumented)
+export type SchemaGeneratorConfig = {
+    connectionString: string;
+    out: string;
+};
+
+// @public (undocumented)
+export type SQLDataSourceConfig = {
+    engine: SQLEngine;
+    username: string;
+    password: string;
+    host: string;
+    database: string;
+    port: number;
+};
+
+// @public (undocumented)
+export type SQLEngine = MySQLEngine | PostgresSQLEngine;

```

Comment:
```suggestion
export type SQLEngine = 'mysql' | 'postgresql';

```

and remove `MySQLEngine ,  PostgresSQLEngine` types.
there doesn't seem to be any api exposed that would take sub type. So let's inline it and make public api smaller.